licenserc.toml

```diff
@@ -0,0 +1,29 @@
+additionalHeaders = ["scripts/custom-header-style.toml"]
+
+headerPath = "scripts/license-header.txt"
+
+includes = [
+    "*.c",
+    "*.cc",
+    "*.h",
+    "*.js",
+    "*.pkl",
+    "*.py",
+    "*.rs",
+    "*.sh",
+    "*.swift",
+    "Cargo.toml",
+    "PklProject",
+]
+
+excludes = []
```

Comment:
This repo has a mixture of generated code and hand-written code.

Let's exclude the generated code, as I don't think this is very common practice.

I think this should do it:

```
excludes = [
    "bindings/*",
    "parser.c",
    "src/tree_sitter/*"
]
```